parm/archive/gefs_arcdir.yaml.j2

```
@@ -21,7 +21,7 @@
 {% if RUN == "gefs" %}
```

To add the lines below, `REPLAY_ICS` and `OFFSET_START_HOUR` will need to be added to `keys` in `scripts/exglobal_archive.py`, but this is already being done in Anton's PR. 

```suggestion
{% if REPLAY_ICS == "YES" %}
    {% set ofst_hr = OFFSET_START_HOUR %}
{% else %}
    {% set ofst_hr = FHMIN_GFS %}
{% endif %}

{% if RUN == "gefs" %}
```